config.cmake

```diff
@@ -58,7 +58,9 @@ HHVM_EXTENSION(mongodb
  libmongoc/src/mongoc/mongoc-gridfs-file.c libmongoc/src/mongoc/mongoc-gridfs.c
  libmongoc/src/mongoc/mongoc-host-list.c
  libmongoc/src/mongoc/mongoc-index.c libmongoc/src/mongoc/mongoc-init.c
- libmongoc/src/mongoc/mongoc-list.c libmongoc/src/mongoc/mongoc-log.c
+ libmongoc/src/mongoc/mongoc-list.c
+ libmongoc/src/mongoc/mongoc-linux-distro-scanner.c
+ libmongoc/src/mongoc/mongoc-log.c
```

Comment:
No, they were simply broken off at 80 chars originally, but other files were added later messing that all up.